.github/workflows/deploy-preview.yml

```diff
@@ -21,46 +19,40 @@ jobs:
       - uses: pnpm/action-setup@v2
         with:
           version: 8
-      - name: Use Node.js 18
-        uses: actions/setup-node@v4
+      - uses: actions/setup-node@v4
         with:
           node-version: 18
           cache: "pnpm"
-      - name: Install dependencies
-        run: pnpm install
 
-      - name: Get branch name
+      - run: pnpm install
+
+      - name: Get cleaned up branch name
```

Comment:
It wasn't clear to me why we needed this action, so naming it makes sense to me. Not sure this name is ideal though.

Reply:
How about "Get git branch name"

Reply:
```suggestion
      - name: Get git branch name
```